kitsune/customercare/forms.py

```diff
@@ -0,0 +1,75 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _lazy
+
+from zenpy.lib.api_objects import Ticket
+
+TICKET_FORM_ID = 360000417171
+
+# See docs/zendesk.md for details about getting the valid choice values for each field:
+
+PRODUCT_FIELD_ID = 360047198211
+PRODUCT_OTHER_VALUE = "product_other"
+
+CATEGORY_FIELD_ID = 360047206172
+CATEGORY_CHOICES = [
+    ("payment___billing", _lazy("Payment & Billing")),
```

Comment:
I have mixed feelings about all of these choices here. The feel disconnected from the rest of the platform.